charts/akeyless-api-gateway/README.md

```diff
@@ -114,14 +114,15 @@ The following table lists the configurable parameters of the API Gateway chart a
-| `cachingConf.proActiveCaching.dumpInterval`              | To ensure service continuity in case of power cycle and network outage secrets will be backed up periodically per backup interval.                                                                                              | `nil`                                                          |
-
+| Parameter                                             | Description                                                                                                                                                                                                                           | Default |
+|-------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------|
+| `cachingConf.enabled`                                 | Should Caching be enabled                                                                                                                                                                                                             | `false` |
+| `cachingConf.cacheTTL`                                | Stale timeout in minutes, cache entries which are not accessed within timeout will be removed from cache                                                                                                                              | `nil`   |
+| `cachingConf.proActiveCaching.enabled`                | Should Proactive Caching be enabled                                                                                                                                                                                                   | `false` |
+| `cachingConf.proActiveCaching.minimumFetchingTime`    | When using Caching or/and Proactive Caching, additional secrets will be fetched upon requesting a secret, based on the requestor's access policy. Define minimum fetching interval to avoid over fetching in a given time frame. name | `nil`   |
+| `cachingConf.proActiveCaching.dumpInterval`           | To ensure service continuity in case of power cycle and network outage secrets will be backed up periodically per backup interval.                                                                                                    | `nil`   |
+| `cachingConf.useClusterCache`                         | Should cluster caching be enabled                                                                                                                                                                                                     | `false` |
```

Comment:
what is the diffrenece between" cachingConf.enabled" to "cachingConf.useClusterCache"?

Reply:
cachingConf.enabled - related for enable caching mode.
cachingConf.useClusterCache - use sepice cache type - cluster cache which is redis